lib/cudadrv/memory.jl

```diff
@@ -241,7 +249,7 @@ end
 Prefetches memory to the specified destination device.
 """
 function prefetch(buf::UnifiedBuffer, bytes::Integer=sizeof(buf);
-                  device::CuDevice=CuCurrentDevice(), stream::CuStream=CuDefaultStream())
+                  device::CuDevice=device(), stream::CuStream=stream())
```

Comment:
Still has a stream kw-arg

Reply:
Hm, I guess we should remove the `device` argument there too, then? Or maybe that's pushing it too far... and then we should probably retain most of the `stream` arguments too...